lib/swagger/blocks.rb

```diff
@@ -33,30 +51,44 @@ def self.build_api_json(resource_name, swaggered_classes)
     end
 
     module InternalHelpers
+
       # Return [root_node, api_node_map] from all of the given swaggered_classes.
       def self.parse_swaggered_classes(swaggered_classes)
-        root_nodes = []
-        api_node_map = {}
-        models_nodes = []
+        root_nodes          = []
```

Comment:
Yeah, sorry about that, I should have followed what was there *in situ*; reflex from styles in my regular codebases.